### ISSUE TITLE
Build gradient boosting trees sequentially, with a smaller learning rate

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#262]
and stop whenever we hit validation accuracy max. 

should be pretty easy to do for sklearn, might even be possible to do for xgboost and lightgbm.

**Comment by ClimbsRocks** [comment]
done for sklearn. opening separate requests for xgb and lgb, since they take different approaches.
